boards/arm/bbc_microbit_v2/bbc_microbit_v2.dts

```diff
@@ -113,6 +113,14 @@
 	status = "okay";
 };
 
+&pwm1 {
+	/* buzzer */
+	status = "okay";
```

Comment:
it shouldn't be on by default and only enabled by the overlay.

Reply:
Unlike microbit v1, v2 has an on-board speaker, so I think it makes sense to enable the PWM for driving it at the board level.

But the documentation of the sound sample should be updated, as for microbit v2 it is not required to connect any additional hardware:
https://github.com/zephyrproject-rtos/zephyr/blob/04de43b1a119665532b6b2a0c6bbcbefd47eb7d6/samples/boards/bbc_microbit/sound/README.rst#L9-L17

Also this line from `prj.conf` should be removed (it is redundant for v1, and for v2 it generates a Kconfig warning):
https://github.com/zephyrproject-rtos/zephyr/blob/04de43b1a119665532b6b2a0c6bbcbefd47eb7d6/samples/boards/bbc_microbit/sound/prj.conf#L5

Reply:
> So, should I leave the `status = "okay"`, @kartben @anangl ?

I'm okay with leaving it here.